---
 configure.ac |   18 	2 +	16 -	0 !
 1 file changed, 2 insertions(+), 16 deletions(-)

Index: cfengine-2.2.9/configure.ac
===================================================================
--- cfengine-2.2.9.orig/configure.ac	2009-08-04 12:01:19.000000000 -0400
+++ cfengine-2.2.9/configure.ac	2009-08-04 12:02:00.000000000 -0400
@@ -19,20 +19,6 @@ AM_INIT_AUTOMAKE(cfengine, 2.2.9)   dnl 
 AM_CONFIG_HEADER(src/conf.h)
 
 #
-# Add to the default list of places in LDFLAGS to compensate for
-# ... the configure default value of LIBS on some systems
-#
-for x in /usr/lib64 /usr/local/gnu/lib /usr/local/gnulib /opt/dce/lib /sw/lib
-do
-  if test -d "$x"; then
-    y=`expr " $LDFLAGS " : ".* -L$x "`
-    if test $y -eq 0; then
-      LDFLAGS="$LDFLAGS -L$x"
-    fi
-  fi
-done
-
-#
 # Add to the default list of places in CPPFLAGS to match LDFLAGS above
 # 
 for x in /usr/local/gnu/include /opt/dce/include /sw/include
@@ -148,7 +134,7 @@ else
 	    test "x$BERKELEY_DB_LIB" != "x" && break
             if test -f "$d/include/db.h"; then
                 if test "x$d" != "x/usr"; then
-                    BERKELEY_DB_LDFLAGS="-L$d/lib64 -L$d/lib"
+                    BERKELEY_DB_LDFLAGS="-L$d/lib"
                     BERKELEY_DB_CFLAGS="-I$d/include"
                 fi
                 BERKELEY_DB_LIB="-ldb"
@@ -292,7 +278,7 @@ else
     if test x"$OPENSSL_LIB_DIR" = xdefault ; then
         for d in $prefix /opt/ssl /usr/local/ssl /usr/local /usr; do
             if test -f "$d/include/openssl/opensslv.h"; then
-                OPENSSL_LIB_LDFLAGS="-L$d/lib64 -L$d/lib"
+                OPENSSL_LIB_LDFLAGS="-L$d/lib"
                 OPENSSL_LIB_CPPFLAGS="-I$d/include"
                 OPENSSL_LIB_LIB="-lcrypto"
                 AC_MSG_RESULT($d)
