diff -Naur --exclude '*~' cfengine-2.2.9-fix-format-errors/src/cfetool.c cfengine-2.2.9/src/cfetool.c
--- cfengine-2.2.9-fix-format-errors/src/cfetool.c	2009-02-05 21:49:08.000000000 +0100
+++ cfengine-2.2.9/src/cfetool.c	2009-02-05 21:56:03.000000000 +0100
@@ -3172,7 +3172,7 @@
    {
    snprintf(OUTPUT, CF_BUFSIZE, "Couldn't create database handle\n");
    fprintf(stderr, "db_create: %s\n", strerror(errno));
-   fprintf(stderr, OUTPUT);
+   fprintf(stderr, "%s", OUTPUT);
    return -1;
   }
 
@@ -3184,7 +3184,7 @@
    {
    snprintf(OUTPUT, CF_BUFSIZE, "Couldn't open average database %s\n", AVDB);
    fprintf(stderr, "db_open: %s\n", strerror(errno));
-   fprintf(stderr, OUTPUT);
+   fprintf(stderr, "%s", OUTPUT);
    return -1;
    }
 
diff -Naur --exclude '*~' cfengine-2.2.9-fix-format-errors/src/env_monitor.c cfengine-2.2.9/src/env_monitor.c
--- cfengine-2.2.9-fix-format-errors/src/env_monitor.c	2009-02-05 21:49:08.000000000 +0100
+++ cfengine-2.2.9/src/env_monitor.c	2009-02-05 21:53:38.000000000 +0100
@@ -885,7 +885,7 @@
          {
          CfLog(cfloginform,OUTPUT,"");
          }
-      Verbose(OUTPUT);
+      Verbose("%s", OUTPUT);
       
       snprintf(OUTPUT,CF_BUFSIZE,"LDT_BUF (%s): Rot ",OBS[i][0]);
 
@@ -917,7 +917,7 @@
          {
          CfLog(cfloginform,OUTPUT,"");
          }
-      Verbose(OUTPUT);
+      Verbose("%s", OUTPUT);
 
       if (THIS[i] > av.Q[i].expect)
          {
